will.planx-pla.net/portal/gitops.json

```diff
@@ -0,0 +1,314 @@
+{
+  "gaTrackingId": "UA-119127212-6",
+  "graphql": {
+    "boardCounts": [
+      {
+        "graphql": "_case_count",
+        "name": "Case",
+        "plural": "Cases"
+      },
+      {
+        "graphql": "_experiment_count",
+        "name": "Experiment",
+        "plural": "Experiments"
+      },
+      {
+        "graphql": "_aliquot_count",
+        "name": "Aliquot",
+        "plural": "Aliquots"
+      }
+    ],
+    "chartCounts": [
+      {
+        "graphql": "_case_count",
+        "name": "Case"
+      },
+      {
+        "graphql": "_experiment_count",
+        "name": "Experiment"
+      }
+    ],
+    "projectDetails": "boardCounts"
+  },
+  "components": {
+    "appName": "The Michael Commons Portal",
+    "index": {
+      "introduction": {
+        "heading": "Michael Commons",
```

Comment:
It will just be cosmetic, but you can change the references to Michaels commons here.